.changeset/modern-eggs-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clack/prompts': minor
```

Comment:
Can we change these to `patch`. The only reason I don't want it going past `v1` until we knock all the boxes for the roadmap.

Reply:
Actually it can go beyond one digit, like [ni](https://github.com/antfu/ni) package that is on `v0.21.6`